<launch>
<node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
    <param name="framerate" value="10" />
  </node>
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node>

  <node name="image_view_face" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/face_detection/face_image"/>
    <param name="autosize" value="true" />
  </node>
  
<rosparam file="/home/jsaraydaryan/ros_robotcupathome_ws/src/people_management/ros_face_recognition/config/common_face.yaml" />
   <node pkg="people_face_identification" type="PeopleFaceIdentification_simple.py" name="PeopleFaceIdentification"  output="screen" /> 

</launch>